DevCycle/DVCVariable.swift

```diff
@@ -45,21 +45,23 @@ public class DVCVariable<T> {
     
     func update(from variable: Variable) {
         if let value = variable.value as? T {
+            if let handler = self.handler,
+               !isEqual(self.value, variable.value)
+            {
+                handler(value)
+            }
             self.value = value
         } else {
             Log.warn("Variable \(variable.key) does not match type of default value \(T.self))")
         }
         self.type = variable.type
         self.evalReason = variable.evalReason
         self.isDefaulted = false
```

Comment:
pretty sure we want these values set b4 we call `handler()`

Reply:
@jsalaber this one still needs to be fixed.

Reply:
fixed